platform/root/index.ts

```diff
@@ -36,9 +36,16 @@ export class Root {
   }
 
   async start() {
-    const loggingConfig$ = this.configService.atPath('logging', LoggerConfig);
-
-    this.loggerService.upgrade(loggingConfig$);
+    try {
+      const loggingConfig$ = this.configService.atPath('logging', LoggerConfig);
+      this.loggerService.upgrade(loggingConfig$);
+    } catch (e) {
+      // This specifically console.logs because we were not able to configure
+      // the logger.
+      console.log('Configuring logger failed with the following errors:');
+      console.log(e.message);
+      this.shutdown(e);
```

Comment:
__nit:__  I believe we should return here, right? Also maybe we should ` return shutdown(err);` here and below so that promise-based result of `shutdown` is not ignored?